scripts/make_zephyr_sdk.sh

```diff
@@ -21,11 +21,25 @@ fi
 root_dir=$(dirname $0)/..
 sdk_version=$(cat $root_dir/VERSION)
 machine=$1
+
+case "$(uname -s)" in
+   Darwin)
+     os=macosx
```

Comment:
would probably be better to name it `macos` since "Mac OS X" nomenclature is no longer used (and more importantly, it isn't "X," aka. 10, anymore).